tests/integration/test_record.py

```diff
@@ -347,3 +399,27 @@ def test_records_can_be_merged(app, records_not_merged_in_marcxml):
     with app.test_client() as client:
         assert client.get('/api/literature/111').status_code == 200
         assert client.get('/api/literature/222').status_code == 301
+
+
+def test_records_propagate_links_when_merged(app, records_to_be_merged):
+    before_record_update.disconnect(check_if_record_is_going_to_be_deleted)
```

Comment:
This is why receivers are a bad idea: with just two receivers that listen for data changes on the same attributes, you run into these kind of issues, where the two receivers conflict, and you have to disable one to ensure that the other one can do its job properly.